controllers/keda/scaledjob_controller.go

```diff
@@ -101,7 +101,8 @@ func (r *ScaledJobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	// ensure Status Conditions are initialized
 	if !scaledJob.Status.Conditions.AreInitialized() {
 		conditions := kedav1alpha1.GetInitializedConditions()
-		if err := kedacontrollerutil.SetStatusConditions(ctx, r.Client, reqLogger, scaledJob, conditions); err != nil {
+		if err := util.SetStatusConditions(ctx, r.Client, scaledJob, conditions); err != nil {
+			reqLogger.Error(err, "Failed to patch Objects Status with Conditions")
```

Comment:
please start the error message with lowercase (the same for the other error messages in this PR)

Reply:
I think I fixed all of them in the new diff.